_layouts/default.html

```diff
@@ -21,4 +21,4 @@
     {% include footer.html %}
 
   </body>
-</html>
+</html>
```

Comment:
Is this newline removal really needed? `_post.html` too?

Reply:
We are currently outputting line 185 a blank line as extra to the browser.  
Larger page size for everyone to download, more cpu cycles etc
Minor issue I picked up on.

Reply:
One newline (well, actually two newlines) is less than an error, I think. But OK, I understand your intention.